### PR TITLE
Automatically show correct year in about box

### PR DESCRIPTION
**Pull request by albert-github**
Automatically show copyright year of build in the about box

### CODE REVIEW

File: addon/doxywizard/doxywizard.cpp [hunk 1]
```diff
@@ -263,6 +264,12 @@ void MainWindow::manual()
 
 void MainWindow::about()
 {
+  auto now = std::chrono::system_clock::now();
+  std::time_t time = std::chrono::system_clock::to_time_t(now);
+  auto current = *localtime(&time);
+  char tmp[10];
+  sprintf(tmp,"%d",current.tm_year+1900);
```
Comment (doxygen) [inline addon/doxywizard/doxywizard.cpp:271]: Since we have a dependency on Qt here, I think it better to use `int year = QDate::currentDate().year();`

Comment (albert-github) [reply addon/doxywizard/doxywizard.cpp]: I wanted to have it as close as possible to the version used in the doxygen main program so I used the construct as proposed, but I see the point. Adjusted.

**Comment by doxygen** [comment]
even the `tmp` variable was not needed.

**Comment by albert-github** [comment]
Indeed as the construct changed from one line multiple lines and the `<<` operator also supports the `int` like a `%d`.

**Pull request merged**